vitrina/datasets/models.py

```diff
@@ -121,6 +121,27 @@ def get_tag_list(self):
         return str(self.tags).replace(" ", "").split(',') if self.tags else []
 
 
+class DatasetMembers(models.Model):
+    CREATOR = 'dct:creator'
+    CONTRIBUTOR = 'dct:contributor'
+    PUBLISHER = 'dct:publisher'
+
+    ROLE_CHOICES = (
+        (CREATOR, _('Creator')),
+        (CONTRIBUTOR, _('Contributor')),
+        (PUBLISHER, _('Publisher')),
+    )
+    user = models.ForeignKey(User, models.DO_NOTHING, blank=True)
+    organization = models.ForeignKey(Organization, models.DO_NOTHING)
+    role = models.CharField(max_length=255, choices=ROLE_CHOICES)
+    created = models.DateTimeField(null=True, auto_now_add=True)
+    contact = models.BooleanField(null=True)
```

Comment:
```suggestion
    contact = models.BooleanField(default=False)
```

I don't see why we need a third NULL option, a member can be contacted or not, there is no need for a third option.